packages/swirl-components/src/components/swirl-resource-list-item/swirl-resource-list-item.stories.ts

```diff
@@ -30,7 +30,16 @@ const Template = (args) => {
   const element = generateStoryElement("swirl-resource-list-item", args);
 
   element.innerHTML = `
-      <swirl-avatar label="Jane Doe" src="https://avatars.dicebear.com/api/adventurer-neutral/a.svg?size=144" slot="media"></swirl-avatar>
+      <swirl-avatar
+        slot="media"
+        label="Jane Doe"
+        src="https://avatars.dicebear.com/api/adventurer-neutral/a.svg?size=144"></swirl-avatar>
+
+      <swirl-stack slot="badges" align="center" orientation="horizontal" spacing="8">
+        <swirl-icon-notifications-off size="16"></swirl-icon-notifications-off>
+        <swirl-badge label="1" size="s"></swirl-badge>
+      </swirl-stack>
+
   `;
 
   control.slot = "control";
```

Comment:
**question:** In the storybook example, the popover jumps to the top-left of the page as soon as I want to mouse over the popover to click on an action. Is this a Storybook specific behavior only? 

<img width="1503" alt="Screenshot 2023-08-18 at 13 14 09" src="https://github.com/getflip/swirl/assets/142502606/21f6c553-bc33-4345-b589-a633c245d4bb">

Reply:
Seems like a bug. What browser are you on? I cannot see this behavior in Chrome.

Reply:
I think I found the issue. Can you verify that it is fixed? You might need to clear your cache for Storybook.

Reply:
Looks good now 👍